Tests/CMakeLists.txt

```diff
@@ -8,11 +8,12 @@ set (srcs
 
 include_directories (${include_MAPL_Base} ${include_GMAO_pFIO} ${INC_ESMF} ${include_GMAO_hermes})
 
-ecbuild_add_executable (
-  TARGET ExtDataDriver.x 
-  SOURCES ${srcs}
-  LIBS MAPL_Base GMAO_pFIO ${MKL_LIBRARIES} ${OpenMP_Fortran_LIBRARIES}
-  )
+#add_executable (ExtDataDriver.x ${srcs})
+ecbuild_add_executable (TARGET ExtDataDriver.x SOURCES ${srcs})
 
+#install(PROGRAMS ExtDataDriver.x DESTINATION bin)
+
+target_link_libraries (ExtDataDriver.x MAPL_Base GMAO_pFIO ${MKL_LIBRARIES} ${OpenMP_Fortran_LIBRARIES} GMAO_hermes)
```

Comment:
Just a question in re CMake. Is there a reason you moved from using bits of `ecbuild_add_executable` to using `target_link_libraries`? @tclune and I have been having fun trying to track down bits of oddities in the ADAS build.